src/components/Utils/resources.js

```diff
@@ -232,3 +235,27 @@ export const ORDER_TEMPLATE = {
   ...BASE_RESOURCE,
   path: `${CONFIG_API}/:{id}`,
 };
+
+export const ORDER_INVOICES = {
```

Comment:
I'm not sure these resources are commonly used

Reply:
My thought was to keep it in one place